docker-compose.yml

```diff
@@ -39,13 +35,13 @@ services:
     image: postgres
     restart: always
     # Set volume so database not lost after shutting down container
-    # volumes:
-    #   - ./postgres-data:/var/lib/postgresql/data
+    volumes:
+      - ./postgres-data:/var/lib/postgresql/data
     # Setup username, password, and database name
     environment:
-      - POSTGRES_USER=gorm
-      - POSTGRES_PASSWORD=gorm
-      - POSTGRES_DB=gorm
+      - POSTGRES_USER=${POSTGRES_USER}
```

Comment:
Might want to update README if new env variables need to be set from the host machine.

Reply:
Good Call